lib/src/widget/nav_bar/widget/nav_bar.dart

```diff
@@ -41,15 +41,26 @@ class _NavBarState extends State<NavBar> implements ConfigUpdater {
 
   @override
   void update(NavBarConfig? config) {
+    assert(
+      config?.type != NavBarType.ignored,
+      'This config should be ignored and should NOT be populated here',
```

Comment:
💅  This config is ignored, so it can not be used to update the NavBar. This is likely a internal Bug.

Reply:
updated